### PR TITLE
Add Extract Activity to processing workflow

### PR DESCRIPTION
**Pull request by djjuhasz**
Use github.com/artefactual-sdps/temporal-activities/archive to extract archives.

### CODE REVIEW

**Comment by jraddaoui** [comment]
Similar feedback to the one I added in https://github.com/artefactual-sdps/enduro/pull/932. Everything else looks good to me, but I'd wait to hear from @sevein.

**Review by sevein — APPROVED** [review]
Nice to see this coming along. LGTM.

File: main.go [hunk 1]
```diff
@@ -377,17 +379,18 @@ func main() {
...
-	Watcher     watcher.Config
-	Pipeline    []pipeline.Config
-	Validation  validation.Config
-	Telemetry   TelemetryConfig
-	Metadata    metadata.Config
+	Verbosity      int
+	Debug          bool
+	DebugListen    string
+	API            api.Config
+	ExtractArchive archive.Config
```
Comment (jraddaoui) [inline main.go:386]: Based on the config changes [here](https://github.com/artefactual-labs/enduro/pull/605/files#diff-cc2139700f8207bdcf502d8c0c22b6f47567663a6f75468cccede2f88310114cR24) this should be `ExtractActivity`.

**Review by jraddaoui — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (djjuhasz) [reply main.go:386]: Hehe, oops. Good catch @jraddaoui. :)

**Review by jraddaoui — APPROVED** [review]
Thanks!

**Pull request merged**